lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -290,3 +290,42 @@ iex> Supervisor.restart_child(App.Supervisor, Counter)
 iex> Counter.get(Counter) # After the restart, this process can be used again
 0
 ```
+
+## Sending unnecessary data
+
+#### Problem
+
+Sending a message to a process can be an expensive operation if it is big enough, as messages will be fully copied to the receiving process, which is both CPU and memory intensive. This is due to Erlang's "share nothing" architecture where each process has its own memory, simplifying and speeding up garbage collection.
```

Comment:
Rephrased to have shorted sentences (easier to read) + less "it"s so that it's easier to follow.

```suggestion
Sending a message to a process can be an expensive operation if the message is big enough. That's because that message will be fully copied to the receiving process, which is both CPU and memory intensive. This is due to Erlang's "share nothing" architecture, where each process has its own memory, which simplifies and speeds up garbage collection.
```

Reply:
Thank you, long sentences are one of the banes/improvement points of my writing :'(